### ISSUE TITLE
Expand testing for Renviron as part of persistent compiled Stan models

### ISSUE THREAD

**Issue by medewitt** [epinowcast/epinowcast#433]
**Is your feature request related to a problem? Please describe.**
As part of #416 some testing coverage was degraded due to lack of coverage for `HOME` `.Renviron` files.

**Describe the solution you'd like**
Write additional tests to cover these added gaps. These should run on CI but not on CRAN (i.e., `not_cran()`) to stay compliant with the CRAN policies. 

**Describe alternatives you've considered**
No real alternatives

**Additional context**
None.

**Comment by seabbs** [comment]
@medewitt any thoughts on how we can move forward on this?

**Comment by medewitt** [comment]
Yeah let me remind myself of this and get back to you. I had an idea when we were working on this....
